model/task_lifecycle.go

```diff
@@ -256,11 +256,14 @@ func activatePreviousTask(ctx context.Context, taskId, caller string, originalSt
 	}
 
 	grip.Debug(message.Fields{
-		"ticket":         "EVG-17949",
-		"message":        "stepping back task",
-		"stepback_depth": stepbackDepth,
-		"project_id":     t.Project,
-		"task_id":        t.Id,
+		"ticket":                        "EVG-17949",
```

Comment:
We should update this ticket to be a new ticket if we intend to remove this logging at the end of the project; otherwise we shouldn't be logging at all

Reply:
I'm a little uncertain about the ticket that's now linked here, since EVG-20792 looks like it's about _adding_ logging rather than removing it, but I also don't have context about what the logs to be added in that ticket are for. @hadjri can you weigh in here?

Reply:
Yes, [EVG-20792](https://jira.mongodb.org/browse/EVG-20792) is about adding logging to specifically allow us to observe what next task is being chosen by the bisection algo once it is implemented.

It actually looks like [EVG-17949](https://jira.mongodb.org/browse/EVG-17949) was closed because enforcing a maximum stepback depth is no longer going to be implemented in favor of simply implementing stepback bisection. Given that we should just remove this log entirely.